google/ggrc-core
# GGRC-4014 Tree view is not refreshed in Control's tab after adding a new value into optional fields

# Issue description

Tree view is not refreshed in Control's tab after adding a new value into optional fields. 
Issue is reproducible for the following fields:
Control: Kind/Nature, Type/Means, Frequency, Categories, Assertions fields.
Product: Kind/Type
System: Network Zone
Process: Network Zone

# Steps to test the changes

1. Create a program
2. Create and map a new Control to program
3. In Controls tab set visible fields: e.g. KIND/NATURE
4. Invoke Edit Control modal window and add a value to KIND/NATURE field> Save
5. Look at the tree view: added value is displayed in tree view
6. Refresh the page and Invoke Edit Control modal window once again
7. Add a new value to KIND/NATURE field> Save
8. Look at the tree view: KIND/NATURE is not updated with a new value

**Actual Result:** While filling a value to optional field (Kind/Nature, Fraud Related, Significance, Type/Means, Frequency, Assertions, Categories):
tree view is not refreshed;
previous value is displayed
**Expected Result:** While filling a value to optional field (Kind/Nature, Fraud Related, Significance, Type/Means, Frequency, Assertions, Categories):
tree view should be refreshed;
new value should be displayed.
NOTE: the important thing how to reproduce the Issue is that it is necessary to refresh the page before adding a new value to optional field.

# Solution description

Issue is reproducible for stub objects.
'using' mustache helper is not refresh value in tree view. New tree-field component was created which subscribes on change event and refreshes stub object.

# Sanity checklist




# PR Review checklist
